setup-scripts/health-checker.py

```diff
@@ -26,6 +27,12 @@
 # a lower limit in order to leave some space for additional message text.
 DISCORD_MAX_MESSAGE_LENGTH = 1900
 
+GB = 1 << 20 # converts from KiB to GiB
```

Comment:
I found this line confusing, particularly why does a variable called `GB` convert from KiB to GiB. It made sense after I read `check_disk` but I would rename this variable and/or clarify the comment. Probably just the comment so the following line `50 * GB` continues to make sense.

Reply:
The Python scripts are turning out to be problematic, so we're most probably going to rewrite them in something easier to maintain such as TypeScript. So, for the moment I'll merge this as it and I'll leave some local notes to improve the documentation.